packages/@tinacms/core/src/event.ts

```diff
@@ -47,7 +47,7 @@ export class EventBus {
 }
 
 export class Listener {
-  constructor(private eventPattern: string, private callback: Callback) {}
+  constructor(private matchEventPattern: string, private callback: Callback) {}
```

Comment:
I think I was misunderstood, I didn't mean rename the var. FWIW the old name was better 😅